src/main/kotlin/com/linchpino/core/controller/InterviewTypeAdminController.kt

```diff
@@ -98,6 +103,16 @@ class InterviewTypeAdminController(private val service: InterviewTypeService) {
     @ResponseStatus(HttpStatus.NO_CONTENT)
     @DeleteMapping("/{id}")
     fun deleteInterviewTypeById(@PathVariable id: Long) {
-        service.deleteInterviewType(id)
```

Comment:
I tried catching exception in service layer, but for some strange reason the catch clause could not catch any exception even java.lang.Exception, therefore I moved the exception handling to controller.